### PR TITLE
fix(email): log 'to' recipient alongside smtp message-id

### LINKED ISSUE #2285: log email address alongside email messageId after mail send
For debugging purposes, it would be useful if the smtp `to` were logged alonside `message-id` here (maybe some other metadata):

https://github.com/mozilla/fxa-auth-server/blob/master/lib/senders/email.js#L296-L314

e.g., if I have to ask SES about an email, they want the `message-id`. Without the email address, I can't unambiguously determine that value.

### PR DESCRIPTION
**Pull request by jrgm**
fixes #2285 

r? - @mozilla/fxa-devs

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -2316,15 +2316,6 @@ Returns SMS status for the current user.
   is in the specified country.
   <!--end-query-param-get-smsstatus-country-->
 
-##### Error responses
```
Comment (vladikoff) [inline docs/api.md]: O_o

Comment (rfk) [reply docs/api.md]: yeah, this happened to me as well: https://github.com/mozilla/fxa-auth-server/pull/2278/files#r165863399

Comment (jrgm) [reply docs/api.md]: How did that get removed?

I'll put it back.

**Comment by jrgm** [comment]
put back the auto-magic-remove commit.

**Pull request merged**